### ISSUE TITLE
SCSS checker definition is flawed

### ISSUE THREAD

**Issue by mravery** [flycheck/flycheck#513]
I am running 

Flycheck version: 0.22snapshot (package: 20141026.627)
GNU Emacs 24.4.1 (x86_64-apple-darwin14.0.0, NS apple-appkit-1343.14)

with prelude

Whenever I'm in a SCSS file and do a "save" I get the following message:

```
Checker scss returned non-zero exit code 1, but no errors from output: /Users/avery/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/site_ruby/2.0.0/rubygems/dependency.rb:315:in `to_specs': Could not find 'sass' (>= 0) among 14 total gem(s) (Gem::LoadError)
Checked in 'GEM_PATH=/Users/avery/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/gems/2.0.0:/Users/avery/.gem/ruby/2.0.0', execute `gem env` for more information
    from /Users/avery/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/site_ruby/2.0.0/rubygems/dependency.rb:324:in `to_spec'
    from /Users/avery/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_gem.rb:58:in `gem'
    from /Users/avery/.rvm/gems/ruby-2.0.0-p576/bin/scss:22:in `<main>'
    from /Users/avery/.rvm/gems/ruby-2.0.0-p576/bin/ruby_executable_hooks:15:in `eval'
    from /Users/avery/.rvm/gems/ruby-2.0.0-p576/bin/ruby_executable_hooks:15:in `<main>'

Checker definition probably flawed.
```

The sass gem is already installed. Further, my shell reports the GEM_PATH as `/Users/avery/.rvm/gems/ruby-2.0.0-p576:/Users/avery/.rvm/gems/ruby-2.0.0-p576@global` not what is reported above. It's clear why flycheck can't find `sass`... it's not at the path it's looking for.
